website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -267,61 +271,63 @@ const MessageActions = ({
       <MenuButton>
         <MoreHorizontal />
       </MenuButton>
-      <MenuList>
-        <MenuGroup title={t("reactions")}>
-          <SimpleGrid columns={4}>
-            {["+1", "-1"].map((emoji) => (
-              <EmojiMenuItem key={emoji} emoji={emoji} checked={userEmoji?.includes(emoji)} react={react} />
-            ))}
-          </SimpleGrid>
-        </MenuGroup>
-        <MenuDivider />
-        <MenuItem onClick={onLabel} icon={<ClipboardList />}>
-          {t("label_action")}
-        </MenuItem>
-        <MenuItem onClick={onReport} icon={<Flag />}>
-          {t("report_action")}
-        </MenuItem>
-        <MenuDivider />
-        <MenuItem as={NextLink} href={`/messages/${id}`} target="_blank" icon={<MessageSquare />}>
-          {t("open_new_tab_action")}
-        </MenuItem>
+      <Portal>
```

Comment:
Portal is required here. Having the menu inside the component will make the parent overflow. Even when adding over-flow: auto it still has an ugly scroll bar